src/Catalyst.Abstractions/Kvm/Models/BlockForRpc.cs

```diff
@@ -22,62 +22,60 @@
 #endregion
 
 using System.Collections.Generic;
-using System.Linq;
-using System.Numerics;
+using LibP2P;
 using Nethermind.Core;
 using Nethermind.Core.Crypto;
-using Nethermind.Core.Encoding;
-using Nethermind.Core.Extensions;
 using Nethermind.Core.Json;
 using Nethermind.Dirichlet.Numerics;
 using Newtonsoft.Json;
 
 namespace Catalyst.Abstractions.Kvm.Models
 {
+    [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
     public class BlockForRpc
     {
-        private readonly BlockDecoder _blockDecoder = new BlockDecoder();
+        // private readonly BlockDecoder _blockDecoder = new BlockDecoder();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4872747)